src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -51,8 +54,32 @@ function WorkspaceWorkflowsPage({policy, betas, route}: WorkspaceWorkflowsPagePr
 
     const onPressAutoReportingFrequency = useCallback(() => Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_AUTOREPORTING_FREQUENCY.getRoute(policy?.id ?? '')), [policy?.id]);
 
-    const items: ToggleSettingOptionRowProps[] = useMemo(
-        () => [
+    const authorizedPayerEmail = policy?.reimburserEmail;
+    const displayNameForAuthorizedPayer = useMemo(
+        () => PersonalDetailsUtils.getPersonalDetailByEmail(authorizedPayerEmail ?? '')?.displayName ?? authorizedPayerEmail,
+        [authorizedPayerEmail],
+    );
+
+    const fetchData = useCallback(() => {
+        if (!policy?.id) {
+            return;
+        }
+        Policy.openWorkspaceReimburseView(policy?.id);
+    }, [policy]);
+
+    useEffect(() => {
+        if (isOffline) {
+            return;
+        }
+        fetchData();
+    }, [isOffline, fetchData, reimbursementAccount]);
```

Comment:
I think a better way is to add `fetchData` as a callback to the `useNetwork` hook, like we do [here](https://github.com/Expensify/App/blob/9ce646de8b298c914e13bc56c40daef8afa478be/src/pages/workspace/categories/WorkspaceCategoriesPage.tsx#L63-L69), so it triggers when we reconnect